docker/development/.ruby-version

```diff
@@ -0,0 +1 @@
+3.1.4
```

Comment:
why update in this PR?

Reply:
I can't remember having changed anything in the docker folder. This file should also be removed from this PR.

Reply:
I removed it.